server/src/main/java/keywhiz/KeywhizConfig.java

```diff
@@ -90,7 +89,7 @@
   private String statusCacheExpiry;
 
   @JsonProperty
-  private String rowHmacCheck;
+  private RowHmacCheck rowHmacCheck;
```

Comment:
Jackson supports properties which allow direct mapping from strings to enum values, so we can now store the property natively as an enum.